crates/ruff_linter/src/rules/ruff/rules/used_dummy_variable.rs

```diff
@@ -47,6 +52,16 @@ use crate::{checkers::ast::Checker, renamer::Renamer};
 /// would not shadow any other known variables already accessible from the scope
 /// in which the variable is defined.
 ///
+/// ## Fix safety
+/// This rule's fix is marked as unsafe.
+///
+/// As well as renaming the variable itself, the fix iterates through all references to the
+/// variable and adapts them to become references to the renamed variable. However, some renamings
```

Comment:
How about:

```diff
- /// As well as renaming the variable itself, the fix iterates through all references to the
- /// variable and adapts them to become references to the renamed variable. However, some renamings
+ /// Ruff will attempt to rename the variable and all references to it. However, some renamings
```

Reply:
Hmm... I like the improved concision... but I'm not sure about talking about "renaming references". I feel like, strictly speaking, the binding is renamed, but the references do not have a "name" as such (so they cannot be renamed), they only _point to_ (refer) to a name.

I'll push something that's more concise than what I've got currently, though ;)

Thanks!